packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.vue

```diff
@@ -347,14 +375,16 @@ const selectInputProps = computed(() => {
               </template>
 
               <OnyxSelectOption
-                v-for="option in options"
+                v-for="option in groupOptions"
                 :key="option.value.toString()"
                 v-bind="
                   headlessOption({
                     value: option.value,
                     label: option.label,
                     disabled: option.disabled,
-                    selected: arrayValue.some(({ value }) => value === option.value),
+                    selected:
+                      arrayValue.some((value) => value === option.value) ||
+                      props.modelValue === option.value,
```

Comment:
TODO: need help

this change ewas necessary, else the `test("should pass headless accessibility tests (select only)"` failed on the step `expectToBeSelected` - arrayValue was not yet updated at that time, but modelValue was... 
is there a way to stabilize the playwright test without this redundant workaround here?

Reply:
solution note: in arrayValue, we did not check model value to be undefined but to be falsy. in the test, the ID was 0 so it failed.